src/components/mainNavbar.jsx

```diff
@@ -67,14 +67,14 @@ const MainNavbar = () => {
         <ul className={classes.navContainerList}>
           {isAuth ? (
             <>
-              {navItems.map((item) => {
-                return <li className={classes.navList}>{item}</li>;
+              {navItems.map((item, index) => {
```

Comment:
Why need index?

Reply:
We are using autocomplete attributes in the input fields. Because this file not using the  `<form>` tag to toggle the autocomplete. `autocomplete ` lets web developers specify what if any permission the user agent has to provide automated assistance in filling out form field values, as well as guidance to the browser as to the type of information expected in the field.

Reply:
Every child in a list should have a unique "key" prop.